backends/graphics/openglsdl/openglsdl-graphics.cpp

```diff
@@ -352,37 +348,54 @@ bool OpenGLSdlGraphicsManager::loadVideoMode(uint requestedWidth, uint requested
 	// Fetch current desktop resolution and determining max. width and height
 	Common::Rect desktopRes = _window->getDesktopResolution();
 
-	if (_windowIsMaximized == true) {
+	// Determine current desktop aspect ratio
+	float ratio = (float)desktopRes.width() / (float)desktopRes.height();
```

Comment:
Ha, I see now that it is still used, as you removed the other ratio computed from the computed size instead.
But we could argue that if we need to resize the window because the computed size is too big, then we should probably use either the computed ratio, or the requested ratio, but not the desktop ratio. I think using the requested ratio might make more sense.
So either remove this ratio and add back the other one, or use `float ratio = (float) _lastRequestedWidth / (float) _lastRequestedHeight;`

Reply:
As you proposed below, I reinstated this line.